survey_form/index.html

```diff
@@ -45,6 +45,35 @@
 
     <form class="form">
       <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+
+      <div class="form__item">
+        <label for="name">Name: </label>
+        <input
+          type="text"
+          id="name"
+          class="form__item--input"
+          required
+          placeholder="Enter your name"
+        />
+
+        <label for="email">Email: </label>
+        <input
+          type="email"
+          id="email"
+          class="form__item--input"
+          required
+          placeholder="Enter your correct email"
+        />
+
+        <label for="number">Age: </label>
+        <input
+          type="number"
+          id="number"
+          class="form__item--input"
```

Comment:
Po naniesieniu powyższych zmian, odpowiednią klasą dla inputa będzie coś w stylu: `form-survey__input`.

Głównym `B`lokiem będzie `form-survey`, `E`lementem będzie `input` - klasa `form-survey__input`. Ewentualnym `M`odifikatorem mogłabyć klasa `form-survey__input--numeric`, która modyfikuje `input`, jeśli ten ma typ `number`.

Także pozmieniaj we wszystkich inputach. Prawdopodobnie będziesz potrzebował ostylować jeszcze jeden element - label.